src/styles/tailwind.css

```
@@ -1,7 +1,6 @@
 @import 'tailwindcss';
-@import './prism.css' layer(components);
 
-@config '../../tailwind.config.js';
+@config '../../tailwind.config.cjs';
```

Stylelint (17.12.0)

[error] Unexpected unknown at-rule "`@config`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)